conformance/scenarios/httpgateway/httpgateway.yaml

```diff
@@ -0,0 +1,56 @@
+kind: GatewayClass
```

Comment:
nit: Consider moving `conformance` dir into `test/conformance`. I think this project could potentially have different types of tests in future.